inventory/host_vars/network.yml

```diff
@@ -13,3 +13,7 @@ yamllint:
   ignore: |
     tests/roles/
 role_specific_contributing: include_files/network_contributing.md
+present_badges:
+  - "[![Coverage Status](https://coveralls.io/repos/github/linux-system-roles/network/badge.svg)](https://coveralls.io/github/linux-system-roles/network)"
+  - "[![Code Style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)"
+  - "[![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/linux-system-roles/network.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/linux-system-roles/network/context:python)"
```

Comment:
ok - but note that lgtm.com is deprecated and will be removed at some point https://github.blog/2022-08-15-the-next-step-for-lgtm-com-github-code-scanning/

Reply:
I just used what's in the network role already. We can revisit it later.